Mastodon/Scene/Transition/MediaPreview/MediaHostToMediaPreviewViewControllerAnimatedTransitioning.swift

```diff
@@ -534,7 +534,7 @@ extension MediaHostToMediaPreviewViewControllerAnimatedTransitioning {
         let scaleTransform = CGAffineTransform(scaleX: (itemWidth / size.width), y: (itemHeight / size.height))
         let scaledOffset = transitionItem.touchOffset.apply(transform: scaleTransform)
 
-        let center = transitionView.convert(transitionView.center, to: nil)
```

Comment:
The `center` property of `UIView` is relative to the `superview`'s coordinate system, but since we were calling `convert(_:to:)` on `transitionView` itself, we were implying that `transitionView.center` was relative to `transitionView`'s own coordinate space. This led to a "double offsetting", if you will.

The fix would be either to:
- Call `convert(_:to:)` on `transitioinView.superview`. This requires some ergonomic pain in the code since `superview` is an optional.
- Pass the center of `transitionView`'s `bounds` to the `convert(_:to:)` call so that we eliminate this double conversion. This was a bit easier code-wise here, so that's what I did.